src/therapy/schemas.py

```diff
@@ -258,37 +259,39 @@ class NamespacePrefix(Enum):
     WIKIDATA = "wikidata"
 
 
-# Source to URI. Will use OBO Foundry persistent URL (PURL) or source homepage
-NAMESPACE_TO_SYSTEM_URI: dict[NamespacePrefix, str] = {
-    NamespacePrefix.ATC: "https://www.who.int/tools/atc-ddd-toolkit/atc-classification/",
-    NamespacePrefix.CHEBI: "http://purl.obolibrary.org/obo/chebi.owl",
-    NamespacePrefix.CHEMBL: "https://www.ebi.ac.uk/chembl/",
-    NamespacePrefix.CHEMIDPLUS: "https://pubchem.ncbi.nlm.nih.gov/source/ChemIDplus",
-    NamespacePrefix.CASREGISTRY: "https://pubchem.ncbi.nlm.nih.gov/source/ChemIDplus",
-    NamespacePrefix.CVX: "https://www2a.cdc.gov/vaccines/iis/iisstandards/vaccines.asp?rpt=cvx",
-    NamespacePrefix.DRUGBANK: "https://go.drugbank.com",
-    NamespacePrefix.DRUGCENTRAL: "https://drugcentral.org",
-    NamespacePrefix.DRUGSATFDA_ANDA: "https://www.fda.gov/drugs/types-applications/abbreviated-new-drug-application-anda",
-    NamespacePrefix.DRUGSATFDA_NDA: "https://www.fda.gov/drugs/types-applications/new-drug-application-nda",
-    NamespacePrefix.HEMONC: "https://hemonc.org",
-    NamespacePrefix.INCHIKEY: "https://www.chemspider.com",
-    NamespacePrefix.IUPHAR_LIGAND: "https://www.guidetopharmacology.org/GRAC/LigandListForward",
-    NamespacePrefix.GUIDETOPHARMACOLOGY: "https://www.guidetopharmacology.org/GRAC/LigandListForward",
-    NamespacePrefix.MMSL: "https://www.nlm.nih.gov/research/umls/rxnorm/sourcereleasedocs/mmsl.html",
-    NamespacePrefix.MSH: "https://id.nlm.nih.gov/mesh/",
-    NamespacePrefix.NCIT: "http://purl.obolibrary.org/obo/ncit.owl",
-    NamespacePrefix.NDC: "https://dps.fda.gov/ndc",
-    NamespacePrefix.PUBCHEMCOMPOUND: "https://pubchem.ncbi.nlm.nih.gov/docs/compounds",
-    NamespacePrefix.PUBCHEMSUBSTANCE: "https://pubchem.ncbi.nlm.nih.gov/docs/substances",
-    NamespacePrefix.RXNORM: "https://www.nlm.nih.gov/research/umls/rxnorm/index.html",
-    NamespacePrefix.SPL: "https://www.fda.gov/industry/fda-data-standards-advisory-board/structured-product-labeling-resources",
-    NamespacePrefix.UMLS: "https://www.nlm.nih.gov/research/umls/index.html",
-    NamespacePrefix.UNII: "https://precision.fda.gov/uniisearch",
-    NamespacePrefix.UNIPROT: "https://www.uniprot.org",
-    NamespacePrefix.USP: "https://www.usp.org/health-quality-safety/compendial-nomenclature",
-    NamespacePrefix.VANDF: "https://www.nlm.nih.gov/research/umls/sourcereleasedocs/current/VANDF",
-    NamespacePrefix.WIKIDATA: "https://www.wikidata.org",
-}
+# Source to URI. Will use system prefix URL, OBO Foundry persistent URL (PURL) or source homepage
+NAMESPACE_TO_SYSTEM_URI: MappingProxyType[NamespacePrefix, str] = MappingProxyType(
```

Comment:
@jsstevenson Lmk if there are any others we should be bringing to Alex to talk about on Monday

Reply:
I don't think so. none of these sources are particularly semantically rich so I think what you have is good

Reply:
Thanks for checking!